recipe-client-addon/lib/SandboxManager.jsm

```diff
@@ -65,6 +65,43 @@ this.SandboxManager = class {
       }
     });
   }
+
+  /**
+   * Wraps a function that returns a Promise from a privileged (e.g. chrome)
```

Comment:
English/Latin nitpickery nit: I think you mean "i.e." ("id est", so "that/it is"), not "e.g." ("exempli gratia", so "for example"). In this case, 'privileged' and 'chrome' are the same thing, so one is not an example of the other.